defaults/main.yml

```diff
@@ -69,13 +69,6 @@ gitlab_enable_pages: '{{ gitlab_version | version_compare("8.17",
 # Application FQDN and DNS addresses [[[
 # --------------------------------------
 
-# .. envvar:: gitlab__fqdn [[[
-#
-# The Fully Qualified Domain Name of the GitLab application. This address is
-# used to configure the webserver frontend.
-gitlab__fqdn: 'code.{{ gitlab_domain }}'
-
-                                                                   # ]]]
```

Comment:
Done. I know about the two vars, I was just not sure because `gitlab__fqdn` was not used :)